gleam.toml

```diff
@@ -1,6 +1,6 @@
 name = "todomvc"
 version = "0.1.0"
-gleam = ">= 0.32.0"
+gleam = ">= 1.2.0"
```

Comment:
Revert this please, the required Gleam version has not increased as far as I can see.